src/acl.c

```diff
@@ -872,6 +872,7 @@ void ACLAddAllowedFirstArg(user *u, unsigned long id, const char *sub) {
  *         almost surely an error on the user side.
  * ENODEV: The password you are trying to remove from the user does not exist.
  * EBADMSG: The hash you are trying to add is not a valid hash.
+ * ECHILD: Attempt to allow a specific first argument of a subcommand
```

Comment:
i think you may be missing an update to redis.conf?
and maybe also to the redis-doc repo.
please look into it.

Reply:
this only affects the error reply of `ACL SETUSER`
the error replies are not documented in redis.conf nor in redis-docs